build/building/env-vars.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Environment variables for Docker Build
+description: Use environment variables to configure builds
+keywords: docker build, buildx, buildkit, env, environment variables, config
+---
+
+You can set the following environment variables to enable, disable, or change
+the behavior of features related to building:
+
+| Variable                                                          | Type              |
+| ----------------------------------------------------------------- | ----------------- |
+| [BUILDX_BUILDER](#buildx_builder)                                 | String            |
+| [BUILDX_CONFIG](#buildx_config)                                   | String            |
+| [BUILDX_EXPERIMENTAL](#buildx_experimental)                       | Boolean           |
+| [BUILDX_GIT_INFO](#buildx_git_info)                               | Boolean           |
+| [BUILDX_GIT_LABELS](#buildx_git_labels)                           | String \| Boolean |
+| [BUILDX_NO_DEFAULT_ATTESTATIONS](#buildx_no_default_attestations) | Boolean           |
+| [BUILDX_NO_DEFAULT_LOAD](#buildx_no_default_load)                 | Boolean           |
+
+You can express Boolean values for environment variables in different ways. For
+example, `true`, `1`, and `T` all evaluate to true. Evaluation is done using the
+`strconv.ParseBool` function in the Go standard library. See
+[reference documentation](https://pkg.go.dev/strconv#ParseBool) for details.
+
+## BUILDX_BUILDER
+
+Overrides the configured builder instance. Same as the `docker buildx --builder`
+CLI flag.
+
+Usage:
+
+```console
+$ export BUILDX_BUILDER=my-builder
+```
+
+## BUILDX_CONFIG
+
+You can use `BUILDX_CONFIG` to specify the directory containing the BuildKit
+configuration file you want to use. Docker uses the following lookup order for
+the configuration file:
```

Comment:
This is not just for the BuildKit configuration. This folder also contains the buildx store (builder/nodes configuration), debugging data (logs, pid files):

```
$ tree ~/.docker/buildx
├── activity
│   ├── builder
│   ├── builder2
│   ├── buildkit0106
│   ├── buildkit093
│   ├── default
│   ├── docker2010
│   └── docker2300
├── controller
│   └── shared
│       └── buildx.f2f69fb05c9e70329534cd095c7b09c9e1c97446.log
├── current
├── defaults
└── instances
    ├── builder
    ├── builder2
    ├── buildkit0106
    ├── buildkit093
    └── remote-builder

5 directories, 14 files
```

Reply:
OK!

Reply:
but could also contain a `buildkitd.default.toml` file?

Reply:
yes indeed